activiti-cloud-services-modeling/activiti-cloud-services-modeling-service/src/main/resources/schema/process-extensions-schema.json

```diff
@@ -6,47 +6,22 @@
   "definitions": {
     "extensions": {
       "type": "object",
-      "properties": {
-        "properties": {
-          "description": "The extensions properties",
-          "type": "object",
-          "additionalProperties": {
-            "$ref": "#/definitions/properties"
-          }
-        },
-        "mappings": {
-          "type": "object",
-          "additionalProperties": {
-            "type": "object",
-            "propertyNames": {
-              "type": "string",
-              "enum": [
-                "inputs",
-                "outputs"
-              ]
-            },
-            "additionalProperties": {
-              "type": "object",
-              "additionalProperties": {
-                "type": "object",
-                "$ref": "#/definitions/mappings"
-              }
-            }
-          }
-        },
-        "constants": {
-          "type": "object",
-          "additionalProperties": {
-            "type": "object",
-            "additionalProperties": {
-              "type": "object",
-              "$ref": "#/definitions/constants"
-            }
-          }
-        }
+      "additionalProperties": false,
```

Comment:
in the ticket description there is a different type of json , am i missing anything?

Reply:
This is just the first step , this is enhancing the validation that we have for the current extension structure, i have another PR that will upgrade this schema once the FE app is ready.